src/ISA/ISA/ARCTypes/ARCTable.fs

```diff
@@ -1,12 +1,69 @@
 ﻿namespace ISA
 
+
 type ArcTable = 
     {
         Name : string
         ValueHeaders : CompositeHeader []
+        /// Key: Column * Row
         Values : System.Collections.Generic.Dictionary<int*int,CompositeCell>  
     }
 
+    /// Create ArcTable with empty 'ValueHeader' and 'Values' 
+    static member init(name: string) = {
+        Name = name
+        ValueHeaders = [||]
+        Values = System.Collections.Generic.Dictionary<int*int,CompositeCell>()
+    }
+
+    member this.ColumnCount = this.ValueHeaders.Length
```

Comment:
I would replace this with a getter to be sure for future changes. IF we make the table mutable, this construct here would not update with the table.

Reply:
Can you give an example?

Reply:
```fsharp
    member this.ColumnCount 
        with get() = this.ValueHeaders.Length
```

Reply:
never used this before. i fully trust you witth this 👍